### ISSUE TITLE
need environment variable for group/org in deployment yaml

### ISSUE THREAD

**Issue by davco01a** [kabanero-io/kabanero-operator#138]
need environment variable for group/org in deployment yaml

Currently we have this hardcoded in the microservice but it needs to be portable for customer's environments

**Comment by davco01a** [comment]
e.g.
```
metadata:
  group: kabanero

spec:
      containers:
        - env:
            - name: KABANERO_CLI_NAMESPACE
              valueFrom:
                fieldRef:
                  apiVersion: v1
                  fieldPath: metadata.group
```

**Comment by kaczyns** [comment]
@davco01a @tseelbach We need to have a discussion here about specifically what needs to be included in the `Kabanero` document for the CLI microservice to function, and how it should be exposed to the CLI microservice (ie what environment variable names to use).

`KABANERO_CLI_GROUP` was suggested for the org name, but it's not clear what this is used for.  We spoke with @brutif earlier today and it sounded like what is really needed here is a fully qualified team name (which includes both the org and the team).  Lets have a dialog here in this issue about exactly what is needed.

**Comment by kaczyns** [comment]
I am removing the release-0.1 tag based on a conversation with @ralanlittle this afternoon.

**Comment by kaczyns** [comment]
We've come to an agreement that we need the following information in the `Kabanero` document:

1) Github organization that contains the collection repository
2) Github team (in that organization) which should have admin authority to that collection repository
3) The github API url for the github where the collection repository lives (ie.  https://api.github.com)

The github API url can default to https://api.github.com so that if the user is not using GHE, they can just leave it blank.
```yaml
github:
  organization: myorg
  team: myteam
  apiUrl: https://api.github.com
```

**Comment by kaczyns** [comment]
This was fixed in #170.
